Projects/App/Sources/Journeys/TerminationFlowJourney.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import Presentation
+import TerminateContracts
+
+extension AppJourney {
+    @JourneyBuilder
+    static func startTerminationJourney() -> some JourneyPresentation {
+        TerminationFlowJourney.getScreen(for: .navigationAction(action: .openSetTerminationDateScreen))
```

Comment:
@sladan-hedvig Not optimal. Is there any other way we can do this? We need to open some screen in order to get Termination journey started otherwise nothing will show.